{
    "docs": "https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/shebang.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test shebang` [[explain](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/shebang.html)] failed with 1 error:", 
            "output": "badcode.py:1:1: unexpected non-module shebang: b'#!/usr/bin/python'"
        }
    ], 
    "verified": false
}
